swagger.yaml

```diff
@@ -48,6 +51,11 @@ paths:
           type: string
           required: true
           description: component version
+        - in: header
+          name: security-vendor
+          type: string
+          required: false
```

Comment:
It would be good to express vendor value as enum,

`enum: [snyk]`, it would help the API user to understand about possible value to this header field.

Reply:
We are supporting Single Vendor only, No plans in future too of integrating another.
Expecting `string` only in API

Reply:
What is the use of this field if it is going to be a single vendor and it's value is also fixed. IIRC, I raised this point and @SriKrishnaPaparaju mentioned that it will help be useful to distinguish multiple vendors.

Reply:
> What is the use of this field if it is going to be a single vendor and its value is also fixed. 
For Differentiating between Vendor and No Vendor, This Helps in selecting the vendor-specific node.
> IIRC, I raised this point and @SriKrishnaPaparaju mentioned that it will help be useful to distinguish multiple vendors.
Multiple vendors still can be integrated, but for an API call, we support a single vendor response at a time.

Reply:
Thats the reason for asking to add enum. It shows possible accepted values and adds puts constrains on value which you could pass.

Reply:
We already had this discussion with Krishna and Frontend is already inline with this.

Reply:
It is going to be still string, but enum will make it clear that all expected value for this field.